deployment/gcp/cloudbuild_admin_app.yaml

```diff
@@ -0,0 +1,21 @@
+steps:
+  - name: "gcr.io/cloud-builders/docker"
+    entrypoint: "bash"
+    args:
+      - -c
+      - >
+        docker build \
+          --build-arg NEXT_PUBLIC_BACKEND_URL=https://$$DOMAIN/api \
+          --build-arg NEXT_PUBLIC_GOOGLE_LOGIN_CLIENT_ID=$$NEXT_PUBLIC_GOOGLE_LOGIN_CLIENT_ID \
+          -t ${_DOCKER_REGISTRY_DOMAIN}/$PROJECT_ID/${_RESOURCE_PREFIX}/admin_app:latest \
+          .
+    secretEnv: ["NEXT_PUBLIC_GOOGLE_LOGIN_CLIENT_ID", "DOMAIN"]
+availableSecrets:
+  secretManager:
+    - version_name: projects/$PROJECT_ID/secrets/${_RESOURCE_PREFIX}-google-login-client-id/versions/latest
+      env: "NEXT_PUBLIC_GOOGLE_LOGIN_CLIENT_ID"
+    - version_name: projects/$PROJECT_ID/secrets/${_RESOURCE_PREFIX}-domain/versions/latest
+      env: "DOMAIN"
+
+images:
+  - "${_DOCKER_REGISTRY_DOMAIN}/$PROJECT_ID/${_RESOURCE_PREFIX}/admin_app:latest"
```

Comment:
`$PROJECT_ID` is availabe through cloudbuild direclty, rather than from GCP Secret Manager or Github Actions